source/overview/mattermost-cloud-overview.rst

```diff
@@ -0,0 +1,88 @@
+
+Viewing and Managing Billing Details
+------------------------------------
+
+Billing details are presented in **System Console > Billing & Account**.
+
+Subscription
+~~~~~~~~~~~~
+
+The subscription screen outlines the details of the Mattermost Cloud plan, including the cost per user per month and the number of users currently registered in your workspace. Also on this screen is a summary of your last invoice.
+
+Billing history
+~~~~~~~~~~~~~~~
+
+The Billing History screen contains a list of all your monthly invoices and payments. Invoices appear in your billing history immediately after the billing period ends.
+
+Billing periods end at 11:59 P.M. UTC on the last day of the month, at which time subscription costs are calculated and a credit card charge is automatically attempted. It may take up to a day for our billing systems to calculate, charge your credit card, and issue your invoice. Once charges are completed, the invoice will show as *Paid*. The invoice is a downloadable PDF file.
+
+If you have a payment failure, your invoice will show *Payment Failed*. Please review the accuracy of your credit card information in **Payment Information**. Unresolved failed payments may result in a delinquency and an interruption to your subscription.
+
+Company information
+~~~~~~~~~~~~~~~~~~~
+
+The **Company Information** screen contains an area to add your company address where you are physically located. The company address will appear on your invoice if specified. Otherwise the billing address associated with the credit card on file will be used.
+
+Payment information
+~~~~~~~~~~~~~~~~~~~
+
+Credit cards are the only form of payment for customers on a monthly billing cycle. All major credit cards are accepted.
+
+To add a new credit card first delete the saved card, then select **Add a Credit Card** in the top right of the subsequent screen. Mattermost only stores information pertaining to one credit card.
+
+We offer pricing and billing only in U.S. Dollars at this time. Payment will be made in USD converted using the exchange rate from at the time of the transaction.
+
+Sales tax and VAT
+~~~~~~~~~~~~~~~~~
+
+Mattermost reserves the right to assess applicable taxes as required by local law. Depending on location, customers may be charged transaction taxes when purchasing our product. Prices on our website are exclusive of sales tax or VAT.
+
+Failed or late payments
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Administrators are notified immediately of failed payments both in-product with a banner and via email. The notifications provide directions on how to update payment information. Once payment information is updated you are charged right away for the amount owing from the previous billing period. If payment information is not updated and the account remains in arrears, here’s what to expect:
```

Comment:
1. Are payment issues always because of outdated payment information, or could there be other reasons? 
2. Can there be multiple payment methods in an account?